src/commands/suggest.ts

```diff
@@ -0,0 +1,39 @@
+// Codey suggest Command
+import Discord from 'discord.js';
+import { openDB } from '../components/db';
+
+export const suggestCmd = async (message: Discord.Message, args: string[]) => {
+  const db = openDB();
+  const state = 'create';
+  const helpArg = 'help';
+  let words = '';
+  let word = '';
```

Comment:
I don't think you need to define this var here if you're doing `for (word in args)`